index.html

```diff
@@ -2,18 +2,20 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Fork Me! FCC: Test Suite Template</title>
     <link rel="stylesheet" href="css/style.css">
   </head>
 
   <body>
         <main id="main-doc">
-                  <h1>Agile and waterfall approaches</h1d>
+                  <h4>Agile and waterfall approaches</h4d>

```

Comment:
Kindly use <h1> tag for the main heading of your page instead of <h4> tag, Because is the exact heading type for the main heading of any page.

Reply:
well done @Jessekay  on adding the inline comment in the right place and the message is in the same context of the selected block of code 💪, but kindly try your best to be clear when pointing and issue and also when giving suggestions, let the student know which tag you are talking about and give them some convincing arguments why they should make that change also you can provide links to helpful resources about this topic.